src/sections/wrapper.tsx

```diff
@@ -141,18 +146,7 @@ function WrapperPage() {
   const [wxhoprValue, set_wxhoprValue] = useState('');
   const [swapDirection, set_swapDirection] = useState<'xHOPR_to_wxHOPR' | 'wxHOPR_to_xHOPR'>('xHOPR_to_wxHOPR');
   const { address } = useAccount();
-
-  // Fetch balance data
-  const { data: xHOPR_balance } = useBalance({
-    address,
-    token: xHOPR_TOKEN_SMART_CONTRACT_ADDRESS,
-    watch: true,
-  });
-  const { data: wxHOPR_balance } = useBalance({
-    address,
-    token: wxHOPR_TOKEN_SMART_CONTRACT_ADDRESS,
-    watch: true,
-  });
+  const walletBalance = useAppSelector((store) => store.web3.balance);
```

Comment:
Same comment as above, if one of the values comprehended inside the `walletBalance`, would it re-render the whole component?